frontend/src/scenes/sessionRecordings/sessionRecordingsTableLogic.test.tsx

```diff
@@ -46,7 +47,9 @@ describe('sessionRecordingsTableLogic', () => {
                 expectLogic(logic).toMatchValues({ sessionRecordingId: null })
             })
             it('is set by openSessionPlayer and cleared by closeSessionPlayer', async () => {
-                expectLogic(logic, () => logic.actions.openSessionPlayer('abc')).toMatchValues({
+                expectLogic(logic, () =>
+                    logic.actions.openSessionPlayer('abc', RecordingWatchedSource.RecordingsList)
```

Comment:
@mariusandra is there a way to test that the URL was changed through a `actiontToUrl` listener? Would be a nice bonus

Reply:
Not directly unfortunately. `actionToUrl` [just calls](https://github.com/keajs/kea-router/blob/d97ae06c8bec09fdc65cd3db380bc30b91640f18/src/plugin.ts#L128) `router.actions.push(newUrl)`. You can explicitly check that the path was `pushed`/`replaced`, but that's not telling you much more than checking what the new URL is.